src/pages/DetailTodoPage.tsx

```diff
@@ -0,0 +1,27 @@
+import { useLocation, useNavigate } from "react-router-dom";
+import type { Todo } from "../types";
+
+function DetailTodoPage() {
+    const navigate = useNavigate();
+    const location = useLocation();
+
+    const todoId: number = location.state.id;
+
+    const todosString = localStorage.getItem('todos');
+    const todos: Todo[] = todosString ? JSON.parse(todosString) : [];
+    const currentTodo: Todo | undefined = todos.find((todo : Todo) => todo.id === todoId);
+
+    const goList = () => {
+        navigate('/');
+    }
+
+    return (
+        <div>
+            <h1>상세 페이지</h1>
+            <p>내용:{currentTodo?.text} </p>
```

Comment:
```suggestion
            <p>내용:{currentTodo?.text || '-'} </p>
```

undefined일 경우 화면에 출력될 대체 텍스트를 입력해주면 좋을 것 같아요